desktop/apps/consignments2/client/actions.js

```diff
@@ -226,6 +232,50 @@ export function incrementStep () {
   }
 }
 
+export function logIn (values) {
+  return async (dispatch, getState) => {
+    try {
+      const options = {
+        email: values.email,
+        password: values.password,
+        _csrf: sd.CSRF_TOKEN
+      }
+      const user = await request
+                      .post(sd.AP.loginPagePath)
+                      .set('Content-Type', 'application/json')
+                      .set('Accept', 'application/json')
+                      .set('X-Requested-With', 'XMLHttpRequest')
+                      .send(options)
+
+      dispatch(updateUser(user.body.user))
```

Comment:
Instead of using `sd.CURRENT_USER` throughout this app, we set the `user` as part of the state (it can be initially set from `sd.CURRENT_USER` if you are logged in when you view the page). When you do navigate away from the submission flow, the page will refresh and `sd.CURRENT_USER` will be set.